chore: resolve `EditorConfig lint errors`

saurabhraghuvanshii opened the pull request:
Resolves #5156.

## Description

> What is the purpose of this pull request?

This pull request:

-   Resolve lint error acc to Editorconfig

## Related Issues

> Does this pull request have any related issues?

This pull request:

-   resolves #5156  

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [x] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

saurabhraghuvanshii committed:
chore: update lint errors

saurabhraghuvanshii commented:
@Planeshifter One more Thing sir , you should review `tools/docs/jsdoc/templates/html/publish.js` it has too many error on commit .

saurabhraghuvanshii commented:
/stdlib-help

kgryte commented on .devcontainer/README.md:
```diff
@@ -8,7 +8,7 @@ Licensed under the Apache License, Version 2.0 (the "License");
 You may obtain a copy of the License at
 
-   http://www.apache.org/licenses/LICENSE-2.0
+    http://www.apache.org/licenses/LICENSE-2.0
```
All of these license header changes should be reverted. We don't want to change the indentation on these.

kgryte commented on docs/doctest.md:
```diff
@@ -226,14 +226,14 @@ indicates that `foo()` returns a JavaScript `object` with known contents. While
-    'c': [ 7, 8, 9 ],
-    'd': {
-      'beep': 'boop'
+    {
```
This isn't desired. Indentation within these comments should remain two-space.

kgryte commented on docs/gfortran.md:
```diff
@@ -104,37 +104,37 @@ A common scenario in numerical computing is exposing numerical computing librari
-  ! ..
-  add = x + y;
-  return
+    implicit none
```
This is not desired.

kgryte commented on docs/gfortran.md:
```diff
@@ -104,37 +104,37 @@ A common scenario in numerical computing is exposing numerical computing librari
-  ! ..
-  ! External functions:
-  interface
+    implicit none
```
Same comments.

kgryte commented on docs/repl_text.md:
```diff
@@ -86,8 +86,8 @@ The general format of a REPL text is as follows:
     - Nadler, Boaz. 2006. "Design Flaws in the Implementation of the Ziggurat
-      and Monty Python methods (and some remarks on MATLAB randn)." *arXiv*
-      abs/math/0603058 (March).
+        and Monty Python methods (and some remarks on MATLAB randn)." *arXiv*
```
This is not desired.

kgryte commented on docs/repl_text.md:
```diff
@@ -489,12 +489,12 @@ Only include references **if** usage requires citations. If not required, **remo
-      generating exponentially and normally distributed pseudorandom numbers."
-      *Journal of Statistical Computation and Simulation* 86 (7): 1281–94.
-      doi:10.1080/00949655.2015.1060234.
+        and Monty Python methods (and some remarks on MATLAB randn)." *arXiv*
```
Same comments.

kgryte commented on docs/style-guides/git/README.md:
```diff
@@ -58,12 +58,12 @@ The **footer** is **optional**. When present, the footer must conform to the [co
-  │       └─▶︎ Commit Scope: noun describing section of the codebase
-  │
-  └─▶︎ Commit Type: bench|build|chore|deprecate|docs|feat|fix|perf|refactor|remove|revert|style|test|temp
+    │       │             │
```
This is very clearly not desired.

kgryte commented on tools/snippets/cli.md:
```diff
@@ -21,8 +21,8 @@ Usage: TODO [options]
 
-  -h,    --help                Print this message.
-  -V,    --version             Print the package version.
+    -h,    --help                Print this message.
```
This is not desired.

saurabhraghuvanshii replied:
In this file there error of license header missing.

kgryte commented on tools/snippets/test/fixtures/julia/runner.jl:
```diff
@@ -19,7 +19,7 @@
 
 """
-    gen( x, name )
+	gen( x, name )
```
This is not desired within a comment string.

saurabhraghuvanshii replied:
In this it require tab not spaces so putting `tab size: 4` it as that

saurabhraghuvanshii commented:
I do all of this because on running command `ec` for checking `EditorConfig` these file violate indetation some have tab some's have spaces so therefore I changes this

kgryte commented:
@saurabhraghuvanshii Notice our configuration: https://github.com/stdlib-js/stdlib/blob/develop/etc/editorconfig-checker/.editorconfig_checker.json

kgryte commented:
Further notice our config for Markdown: https://github.com/stdlib-js/stdlib/blob/develop/etc/editorconfig-checker/.editorconfig_checker.markdown.json

saurabhraghuvanshii committed:
Merge branch 'stdlib-js:develop' into editorConfig

saurabhraghuvanshii committed:
chore: restore licesen

---
type: pre_push_report
description: Results of running various checks prior to pushing changes.
report:
  - task: run_javascript_examples
    status: na
  - task: run_c_examples
    status: na
  - task: run_cpp_examples
    status: na
  - task: run_javascript_readme_examples
    status: failed
---

saurabhraghuvanshii committed:
chore: remove unwanted change from runner.jl

kgryte commented:
Due to merge conflicts and various undesired changes, I'll go ahead and close this PR.
